package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-plugin-auth-passport-oauth",
-  "version": "4.0.5",
+  "version": "4.0.6",
```

Comment:
You should have used `npm version patch` (see [npm version documentation](https://docs.npmjs.com/cli/version)), because the version number in `package-lock.json` also needs to be updated.

Now you have to update it manually.